packages/dataviews-client/src/resolve-dataviews-generators.ts

```diff
@@ -125,6 +125,7 @@ export function getGeneratorConfig(
           {
             id: generator.id,
             colorRamp: dataview.config?.colorRamp as ColorRampsIds,
+            colorRampWhiteEnd: false,
```

Comment:
I'm having doubts on this. It's `fishing-map` application configuration done in package logic. To me this looks like something that should be configured on the dataview, along with more stuff that is done here depending on if environmental layer or nor.